datastore/kubernetes.go

```diff
@@ -31,6 +31,7 @@ const (
 	KubeStatusPollInterval = 1 * time.Second
 
 	PodProbeInitialDelay             = 3
+	PodProbeTimeoutSeconds           = 5
```

Comment:
Good idea externalizing this
- make the timeout 1 smaller then the probePeriod, just in case :)